pkg/standalone/docker.go

```diff
@@ -122,10 +124,11 @@ func parseDockerError(component string, err error) error {
 }
```

Comment:
do these error conditions remain the same? if so can we rename the function to parseContainerRuntimeError or something like that?

Reply:
Error code remains same see here - https://docs.podman.io/en/latest/markdown/podman-remote.1.html#exit-status